datastore/sqlite_store.go

```diff
@@ -169,6 +169,36 @@ func hashEnrichment(r *driver.EnrichmentRecord) (k string, d []byte) {
 	return "md5", h.Sum(nil)
 }
 
+func getMetadata(ctx context.Context, tx *sql.Tx, kind string, val string) (int64, error) {
+	var metadataID int64
+	s := md5.Sum([]byte(val))
```

Comment:
On the fence about this, it adds space (albeit bounded), but the other option (just dealing with the string) seems very dirty, particularly adding a description text to a unique index.

Reply:
I was going to suggest adding the md5 or sha1 as a `GENERATED` column, but it seems like sqlite doesn't ship with a function for it, so you'd have to supply it as a user function. At that point, precomputing it seems fine.